src/people/widgetViews/BountyHeader.tsx

```diff
@@ -446,7 +451,12 @@ const BountyHeader = ({
                 onChange={onChangeSearch}
                 onKeyUp={(e: any) => {
                   if (e.key === 'Enter' || e.keyCode === 13) {
```

Comment:
Put this functionality in an external function, and pass the function to onKeyUp

Reply:
@elraphty Addressed

Reply:
> @elraphty Addressed

@AbdulWahab3181 handleSearch should receive the event parameter

e.g
```
onKeyUp={handleSearch}

const handleSearch = (e) => {
   if(e) {} 
}
```

Reply:
@elraphty I don't think we need to receive the event parameter. In the onChangeSearch method we sets the [searchText](https://github.com/stakwork/sphinx-tribes-frontend/pull/239/files#diff-f8355a876a6e879aa2e9e4210a2771d0d9e55a556bc54ee5bd8ae3a2d1d4d9daL388) state, which is used inside the getPeopleBounties method [here](https://github.com/stakwork/sphinx-tribes-frontend/blob/master/src/store/main.ts#L857). The handleSearch method is also used [here](https://github.com/stakwork/sphinx-tribes-frontend/pull/239/files#diff-f8355a876a6e879aa2e9e4210a2771d0d9e55a556bc54ee5bd8ae3a2d1d4d9daR402), where we have the search text as an empty string.

Reply:
This works
```sh
const getBounties = () => {
    main.getPeopleBounties({
      page: 1,
      resetPage: true,
      ...checkboxIdToSelectedMap,
      ...checkboxIdToSelectedMapLanguage
    });
  }

  const handleSearch = (e: any) => {
    if (e.key === 'Enter' || e.keyCode === 13) {
      getBounties();
    }
  };

  const clearSearch = () => {
    getBounties()
  }

  let timeoutId;
  const onChangeSearch = (e: any) => {
    ui.setSearchText(e);
    clearTimeout(timeoutId);
    // Set a new timeout to wait for user to pause typing
    timeoutId = setTimeout(() => {
      if (ui.searchText === '') {
        clearSearch();
      }
    }, 1000);
  };

```

Reply:
Addressed